radiation/python/radlw/taumol_gt4py.py

```diff
@@ -0,0 +1,4957 @@
+            id011 = ind1 + 10
+            id101 = ind1
+            id111 = ind1 + 9
+            id201 = ind1 - 1
+            id211 = ind1 + 8
+        else:
+            fk01 = 1.0 - fs1
+            fk11 = fs1
+            fk21 = 0.0
+
+            id001 = ind1
+            id011 = ind1 + 9
+            id101 = ind1 + 1
+            id111 = ind1 + 10
+            id201 = ind1
+            id211 = ind1
+
+        fac001 = fk01 * fac01
+        fac101 = fk11 * fac01
+        fac201 = fk21 * fac01
+        fac011 = fk01 * fac11
+        fac111 = fk11 * fac11
+        fac211 = fk21 * fac11
+
+        for ig in range(ng15):
+            tauself = selffac * (
+                selfref[0, 0, 0][ig, inds]
+                + selffrac * (selfref[0, 0, 0][ig, indsp] - selfref[0, 0, 0][ig, inds])
+            )
+            taufor = forfac * (
+                forref[0, 0, 0][ig, indf]
+                + forfrac * (forref[0, 0, 0][ig, indfp] - forref[0, 0, 0][ig, indf])
+            )
+            n2m1 = ka_mn2[0, 0, 0][ig, jmn2, indm] + fmn2 * (
+                ka_mn2[0, 0, 0][ig, jmn2p, indm] - ka_mn2[0, 0, 0][ig, jmn2, indm]
+            )
+            n2m2 = ka_mn2[0, 0, 0][ig, jmn2, indmp] + fmn2 * (
+                ka_mn2[0, 0, 0][ig, jmn2p, indmp] - ka_mn2[0, 0, 0][ig, jmn2, indmp]
+            )
+            taun2 = scalen2 * (n2m1 + minorfrac * (n2m2 - n2m1))
+
+            taug[0, 0, 0][ns15 + ig] = (
+                speccomb
+                * (
+                    fac000 * absa[0, 0, 0][ig, id000]
+                    + fac010 * absa[0, 0, 0][ig, id010]
+                    + fac100 * absa[0, 0, 0][ig, id100]
+                    + fac110 * absa[0, 0, 0][ig, id110]
+                    + fac200 * absa[0, 0, 0][ig, id200]
+                    + fac210 * absa[0, 0, 0][ig, id210]
+                )
+                + speccomb1
+                * (
+                    fac001 * absa[0, 0, 0][ig, id001]
+                    + fac011 * absa[0, 0, 0][ig, id011]
+                    + fac101 * absa[0, 0, 0][ig, id101]
+                    + fac111 * absa[0, 0, 0][ig, id111]
+                    + fac201 * absa[0, 0, 0][ig, id201]
+                    + fac211 * absa[0, 0, 0][ig, id211]
+                )
+                + tauself
+                + taufor
+                + taun2
+            )
+
+            fracs[0, 0, 0][ns15 + ig] = fracrefa[0, 0, 0][ig, jpl] + fpl * (
+                fracrefa[0, 0, 0][ig, jplp] - fracrefa[0, 0, 0][ig, jpl]
+            )
+
+    with computation(PARALLEL), interval(laytrop, nlay):
+        for ig2 in range(ng15):
+            taug[0, 0, 0][ns15 + ig2] = 0.0
+            fracs[0, 0, 0][ns15 + ig2] = 0.0
+
+
+@stencil(
+    backend=backend,
+    rebuild=rebuild,
+    externals={
+        "nspa": nspa[15],
+        "nspb": nspb[15],
+        "laytrop": indict["laytrop"],
+        "ng16": ng16,
+        "ns16": ns16,
+        "nlay": nlay,
+        "oneminus": oneminus,
+    },
+)
+def taugb16(
```

Comment:
can you add a note to this one about the temporaries only being API fields in this case to work around a bug? that'll help us change it when we have new versions of gt4py to test.

Reply:
Yup will do